spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -66,8 +68,9 @@ def __init__(self, file_path: PathType):
 
         BinDatRecordingExtractor.__init__(self, file_path, sampling_frequency=sampling_frequency,
                                           dtype=dtype, numchan=numchan_from_file)
+        self.set_channel_gains(channel_ids=list(range(numchan_from_file)), gains=gain)
 
-        self._kwargs = {'file_path': str(Path(file_path).absolute())}
+        self._kwargs = dict(file_path=str(Path(file_path).absolute()), gain=1)
```

Comment:
Why `str(Path(file_path)`?

Reply:
This is actually one case I would advocate having the absolute usage, since this is an object that gets saved and might not be re-loaded from the same relative path.

Reply:
ok